lbrynet/lbrynet_daemon/LBRYDaemon.py

```diff
@@ -146,8 +146,8 @@ def __init__(self, root, wallet_type=None):
         self.connected_to_internet = True
         self.connection_problem = None
         self.query_handlers = {}
-        self.git_lbrynet_version = None
-        self.git_lbryum_version = None
+        self.pip_lbrynet_version = None
+        self.pip_lbryum_version = None
```

Comment:
can we stick with grabbing the version from `git` and undo all the changes in this file related to pkg_resources?